src/lib/search-and-replace.js

```diff
@@ -86,24 +114,30 @@ export const searchAndReplace = async (
 	const replacements = flatten( replacementsArr );
 	debug( 'Pairs: ', pairs, 'Replacements: ', replacements );
 
-	// Get a path for a tmp copy of the input file
-	const { inputFile, outputFile } = inPlace
-		? await inPlaceReplacement( filename )
-		: outputFileReplacement( filename );
+	if ( inPlace ) {
+		await confirm(
+			[],
+			'Are you sure you want to run search and replace on your input file? This operation is not reversible.'
+		);
+	}
 
-	console.log( chalk.blueBright( 'Input File Path: ' ), inputFile );
-	console.log( chalk.blueBright( 'Output File Path: ' ), outputFile );
+	const tmpDir = fs.mkdtempSync( path.join( os.tmpdir(), 'vip-search-replace-' ) );
+	debug( `Created a directory to hold temporary files: ${ tmpDir }` );
```

Comment:
Should this be moved into `getReadAndWriteStreams`?

Reply:
done. thanks!